css/syntaxes.json

```diff
@@ -716,6 +767,12 @@
   "step-timing-function": {
     "syntax": "step-start | step-end | steps(<integer>[, [ start | end ] ]?)"
   },
+  "string-token": {
```

Comment:
`string-token` is a token type and out of scope of this dict.
Btw, it has another syntax